### PR TITLE
Add print stylesheet

### LINKED ISSUE #2653: Add print stylesheet
The new theme doesn't have a print stylesheet like the old one, so the Print Lesson Plan action (see single Lesson Plan sidebar) results in a messy print layout:

| Old theme | New theme |
|--------|--------|
| Image | Image |

### PR DESCRIPTION
**Pull request by outdoor2kode**
Resolves #2653

This PR primarily addresses the single lesson plan page. Additionally, some general CSS has been migrated from the Learn 2020 theme `print.scss`. And actually, many of the selectors there are no longer applicable to the new theme. Since there's a CTA for printing on the single lesson plan page, here focusing on that page first. However, though other pages don't have a print CTA, they can still be printed using cmd+P. They can be handled later though.

## Screenshots

|| Before | After |
|-|-|-|
| **Top** | Screenshot 2024-08-02 at 05 25 34 | Screenshot 2024-08-02 at 05 36 31 |
| **Buttom** | image | Screenshot 2024-08-02 at 05 38 31 |

### CODE REVIEW

**Review by adamwoodnz — APPROVED** [review]
LGTM

The only thing I noticed is the large whitespace at the top of page 1, but not sure if we can control that?

**Comment by outdoor2kode** [comment]
53d38b5 Adding `margin-top: 0` removes some whitespace, but not all. 
Not sure what other styles can be used on this off the top of my head.

image

File: wp-content/themes/pub/wporg-learn-2024/src/style/_print.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@media print {
+	// Hide stuff that's not relevant on print media
+	.wp-block-wporg-local-navigation-bar,
+	.wp-block-wporg-site-breadcrumbs,
+	.wporg-learn-sidebar,
+	footer.wp-block-template-part {
```
Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2024/src/style/_print.scss]: Looks like this does the trick

```suggestion
	footer.wp-block-template-part,
	.global-header {
```

Comment (outdoor2kode) [reply wp-content/themes/pub/wporg-learn-2024/src/style/_print.scss]: Ah yeah, it looks nice. 3c6acfb

image

**Review by adamwoodnz — APPROVED** [review]
:shipit:

**Pull request merged**